manager/Makefile

```diff
@@ -50,12 +50,16 @@ prep-test: wait_for_manager
 run-integration-tests: prep-test
 	NO_SIMULATED_PROGRESS=true USE_EXISTING_CONTROLLER=true USE_EXISTING_CLUSTER=true go test ./... $(TEST_OPTIONS) -run TestAPIs -count 1
 
-
 .PHONY: run-notebook-readflow-tests
 run-notebook-readflow-tests: prep-test
 	cd testdata/notebook/read-flow && ./setup.sh
 	NO_SIMULATED_PROGRESS=true USE_EXISTING_CONTROLLER=true USE_EXISTING_CLUSTER=true go test ./... $(TEST_OPTIONS) -run TestS3NotebookReadFlow -count 1
 
+.PHONY: run-deny-by-default-test
+run-deny-by-default-test: prep-test
+	cd testdata/notebook/read-flow && ./setup.sh
```

Comment:
I think we can skip the setup script and create the asset directly in the test code. There are some port-forwards that 1) are not required for the deny test and 2) I am not sure they can be done one after the other.
Another related comment: try to simplify the test. It is a deny, the plotter will never be created, no module will access the data, so creating the s3 service, uploading a file, etc. is an overkill.